test/client/karma.conf.js

```diff
@@ -57,26 +57,22 @@ fs.lstat('node_modules/karma', (err, stats) => {
   console.log('**** Incorrect directory layout for karma self-tests ****')
   console.log(`
     $ npm install
-    
+
     $ npm run init
     # or if you're on Windows
     $ npm run init:windows
-    
+
     $ npm run build
   `)
   process.exit(1)
 })
 
-let browsers = []
+let browsers = ['Chrome']
```

Comment:
Can you help me understand this change? Maybe add some notes on your logic.

Reply:
In the description:
>Run test:client on Chrome for both travis and not travis.

I want the tests on one's own machine to match the ones on travis for at least one browser.